qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -416,16 +421,17 @@ def run_test(self):
         self.sync_all()
 
         #
-        # Test Orchard-only UA with insufficient non-Sprout funds
+        # Test sending Sprout funds to Orchard-only UA
         #
 
+        sproutAddr = self.nodes[0].listaddresses()[0]['sprout']['addresses'][0]
         recipients = [{"address":n0orchard_only, "amount":100}]
         for (policy, msg) in [
             ('FullPrivacy', 'Could not send to a shielded receiver of a unified address without spending funds from a different pool, which would reveal transaction amounts. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` parameter set to `AllowRevealedAmounts` or weaker if you wish to allow this transaction to proceed anyway.'),
             ('AllowRevealedAmounts', 'This transaction would send to a transparent receiver of a unified address, which is not enabled by default because it will publicly reveal transaction recipients and amounts. THIS MAY AFFECT YOUR PRIVACY. Resubmit with the `privacyPolicy` parameter set to `AllowRevealedRecipients` or weaker if you wish to allow this transaction to proceed anyway.'),
             ('AllowRevealedRecipients', 'Could not send to an Orchard-only receiver, despite a lax privacy policy. Either there are insufficient non-Sprout funds (there is no transaction version that supports both Sprout and Orchard), or NU5 has not been activated yet.'),
```

Comment:
Yeah, this message should change. But this loop is the one thing that’s currently failing (with a timeout), so I haven’t yet gotten to it failing because of the error message 😅